napari/layers/shapes/_shape_list.py

```diff
@@ -1113,17 +1121,26 @@ def inside(self, coord):
             Index of shape if any that is at the coordinates. Returns `None`
             if no shape is found.
         """
-        triangles = self._mesh.vertices[self._mesh.displayed_triangles]
-        indices = inside_triangles(triangles - coord)
-        shapes = self._mesh.displayed_triangles_index[indices, 0]
-
-        if len(shapes) == 0:
+        if not self.shapes:
+            return None
+        bounding_boxes = self._bounding_boxes
+        inside = np.all(
```

Comment:
I think this should be renamed "maybe_inside" or "in_bbox", reserving "inside" for shapes that actually contain the coordinate.